dev-python/influxdb-client/influxdb-client-1.16.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="InfluxDB 2.0 Python client library"
+HOMEPAGE="https://github.com/influxdata/influxdb-client-python"
+SRC_URI="https://github.com/influxdata/${PN}-python/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-python-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="ciso extra test"
```

Comment:
Ciso and extra are not know USE flags. Be sure to add them on your metadata.xml after maintainer. Follow my example if you want https://dpaste.com/HRJG4GCAH